### DISCUSSION TITLE
Github Copilot stops working in files with certain words in them.

### DISCUSSION THREAD

**Discussion by AsirShahid** [community/community#33530] [Copilot Conversations]
I was writing a bot that gets Tweets from [this](https://twitter.com/would_it_dong) Twitter account named "would_it_dong" and I noticed that as soon as I typed in the account URL, Copilot stopped giving me suggestions for the rest of the file. In this context "dong" is a Baseball slang term for home runs.

I haven't ran into this issue before and I assumed that this has to do with dong having other meanings. I tried replacing it with some similar words and ran into the same issue.

Has anyone else experienced this issue? If so, how do you fix it?

**Comment by iamcjwilliams** [comment]
@AsirShahid Thank you, the copilot team is currently investigating this issue
